.jenkins/pytorch/build.sh

```diff
@@ -65,13 +65,21 @@ if ([[ "$BUILD_ENVIRONMENT" == *cuda* ]] || [[ "$BUILD_ENVIRONMENT" == *gcc7* ]]
 fi
 
 # Target only our CI GPU machine's CUDA arch to speed up the build
-export TORCH_CUDA_ARCH_LIST=5.2
+export TORCH_CUDA_ARCH_LIST="5.2 6.0"
```

Comment:
Err, we don't want this unconditionally, right? Because the regular CI machines are all 5.2, so the 6.0 build is just a waste of compilation cycles.